server/controllers/api/getlist.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+module.exports = function(app) {
+	
+	var bodyParser = require('body-parser')
+	var jsonParser = bodyParser.json()
+	var urlencodedParser = bodyParser.urlencoded({ extended: false })
+
+	app.use(bodyParser.urlencoded({ extended: false }))
+	app.use(bodyParser.json())
+	
+	app.post('/api/getlist', function (req, res) {
```

Comment:
The following is a good reference for what a REST API would look like:
https://stackoverflow.com/questions/671118/what-exactly-is-restful-programming

The HTTP verb and the URI are important because they tell anyone using the API how it works and what it does.